src/main/webapp/js/application/app-form.js

```diff
@@ -5,6 +5,13 @@ import '../../components/timepicker';
 import sendGetDaysRequest from '../send-get-days-request';
 import sendGetDepartmentVacationsRequest from '../send-get-department-vacations-request';
 
+import count from '../../js/count'
```

Comment:
sollte das hier nicht im bundle unter `.../webapp/bundles/app-form.js` passieren? Analog zu `.../webapp/bundles/sick-note-convert.js` ?

Reply:
Stimmt ist der bisher dafür genutzte Ort. Nur verstehe ich noch nicht ganz wieso wir da 2 Orte haben die sich auf ein JSP beziehen.

Reply:
Ist das dann ready to review? :)